src/Dapr.Client/DaprClientGrpc.cs

```diff
@@ -1015,6 +1015,56 @@ private Autogenerated.StateItem ToAutogeneratedStateItem(StateTransactionRequest
                 throw new DaprException("State operation failed: the Dapr endpoint indicated a failure. See InnerException for details.", ex);
             }
         }
+
+        /// <inheritdoc/>
+        public async override Task<StateQueryResponse<TValue>> QueryStateAsync<TValue>(StateQueryRequest request, CancellationToken cancellationToken = default)
```

Comment:
Let's think about failure modes. If an item fails to decrypt what would a user want to do with partial data? Is there anything useful to be done?

Some options:

- Throw if any items failed and no way to get partial data
- Throw if any items failed and include partial data as part of the exception
- Allow the choice as part of the API (flag)
- Current option: return the data and don't throw on decryption error

Reply:
I think #2 is a good option. I'll implement that.